src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderPhysicalAttributes.kt

```diff
@@ -66,20 +65,4 @@ class OffenderPhysicalAttributes(
   @Column(name = "AUDIT_MODULE_NAME")
   @Generated
   var auditModuleName: String? = null
-
-  fun getHeightInCentimetres() =
-    // Take height in cm if it exists because the data is more accurate (being a smaller unit than inches)
-    if (heightCentimetres != null) {
-      heightCentimetres
-    } else {
-      heightFeet?.let { ((it * 12) + (heightInches ?: 0)) * 2.54 }?.roundToInt()
-    }
-
-  fun getWeightInKilograms() =
-    // Take weight in lb and convert if it exists because the data is more accurate (being a smaller unit than kg). See the unit tests for an example explaining why.
-    if (weightPounds != null) {
-      weightPounds!!.let { (it * 0.453592) }.roundToInt()
-    } else {
-      weightKilograms
-    }
```

Comment:
Moved to the service layer - this logic only makes sense in the context of migration / sync.